install_config/syncing_groups_with_ldap.adoc

```diff
@@ -151,6 +151,13 @@ groupUIDNameMapping:
 ----
 ====
 
+[[LDAP-configuration-with-adctive-directory]]
+=== LDAP Configuration with Active Directory
```

Comment:
It does not matter if its active directory or not, we do not support binary attributes, period.